sensor_msgs/Image road_image
sensor_msgs/Image sign_image
